src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/PanelSummaryReportITest.java

```diff
@@ -0,0 +1,134 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.TestConstants;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+
+@Sql({
+    "/db/truncate.sql",
+    "/db/mod/truncate.sql",
+    "/db/administration/createUsers.sql",
+    "/db/mod/reports/PanelSummaryReportITest_typical.sql"
+})
+@SuppressWarnings("PMD.LawOfDemeter")
+class PanelSummaryReportITest extends AbstractStandardReportControllerITest {
+    @Autowired
+    public PanelSummaryReportITest(TestRestTemplate template) {
+        super(template, PanelSummaryReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getCourtJwt("415");
+    }
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+            .trialNumber("T100000001")
+            .locCode(TestConstants.VALID_COURT_LOCATION)
+            .build());
+    }
+
+    @Test
+    @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+    void positiveTypicalCourt() {
+        testBuilder()
+            .triggerValid()
+            .responseConsumer(this::verifyAndRemoveReportCreated)
+            .assertEquals(getTypicalResponse());
+    }
+
+    @Test
+    @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+    void negativeInvalidPayload() {
+        StandardReportRequest request = getValidPayload();
+        request.setTrialNumber(null);
+        testBuilder()
+            .payload(addReportType(request))
+            .triggerInvalid()
+            .assertInvalidPathParam("trialNumber: must not be blank");
+    }
+
+    @Test
+    @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+    void negativeUnauthorised() {
+        testBuilder()
+            .jwt(getBureauJwt())
+            .triggerInvalid()
+            .assertMojForbiddenResponse("User not allowed to access this report");
+    }
```

Comment:
Need a test like this when a user tries to access a loc code they do not have access too.

I.e 415 user tries to access 249 loc ode

Reply:
added